### PR TITLE
OSDOCS#11296: Doc sweep for Kubernetes 1.29 to 1.30

### PR DESCRIPTION
**Pull request by bergerhoffer**
Version(s):
4.17

Issue:
https://issues.redhat.com/browse/OSDOCS-11296

Link to docs preview:
Changes are all 1:1 find/replaces. Not worth the time spent tracking down the previews, but if you want them, they're all found here: https://gcsweb-ci.apps.ci.l2s4.p1.openshiftapps.com/gcs/test-platform-results/pr-logs/pull/openshift_openshift-docs/81062/pull-ci-openshift-openshift-docs-main-validate-asciidoc/1828872732847116288/artifacts/validate-asciidoc/openshift-docs-preview-comment-pages/artifacts/updated_preview_urls.txt

QE review:
- [x] QE has approved this change.

Additional information:

### CODE REVIEW

**Comment by bergerhoffer** [comment]
@zhouying7780 Can you please QE review? The only updates I'm a little uncertain about are the ones that update image versions (like busybox:1.30)

File: modules/secrets-store-vault.adoc [hunk 1]
```diff
@@ -324,7 +324,7 @@ spec:
     spec:
       terminationGracePeriodSeconds: 0
       containers:
-      - image: registry.k8s.io/e2e-test-images/busybox:1.29-4
+      - image: registry.k8s.io/e2e-test-images/busybox:1.30-4
```
Comment (dfitzmau) [inline modules/secrets-store-vault.adoc:327]: Might be worthwhile getting QE eyes on this value as it differs slightly than other versions.

**Review by dfitzmau — APPROVED** [review]
Discussed using an attribute in a Slack chat, but was dropped based on the fact that these updates are rare.

/remove-label peer-review-in-progress

/label peer-review-done

File: modules/secrets-store-vault.adoc (hunk 1 above)
Comment (bergerhoffer) [reply modules/secrets-store-vault.adoc:327]: Yep definitely - I tagged my QE specifically on these ones, here: https://github.com/openshift/openshift-docs/pull/81062#issuecomment-2317618314

File: cloud_experts_tutorials/cloud-experts-aws-secret-manager.adoc [hunk 2]
```diff
@@ -281,7 +281,7 @@ spec:
         secretProviderClass: "my-application-aws-secrets"
   containers:
   - name: my-application-deployment
-    image: k8s.gcr.io/e2e-test-images/busybox:1.29
+    image: k8s.gcr.io/e2e-test-images/busybox:1.30
```
Comment (zhouying7780) [inline cloud_experts_tutorials/cloud-experts-aws-secret-manager.adoc:284]: yes, busybox  image with tag `1.30` can't be available. also for tag `1.29` , maybe we don't need use tag.

Comment (bergerhoffer) [reply cloud_experts_tutorials/cloud-experts-aws-secret-manager.adoc]: Per our slack conversation, I've reverted the busybox changes to stay at 1.29. Let me know how it looks now!

**Pull request merged**